Dockerfile

```diff
@@ -113,11 +117,19 @@ ADD src/anbox-container-manager.service /lib/systemd/system/anbox-container-mana
 RUN systemctl enable anbox-container-manager
 ADD src/unsudo /usr/local/bin
 ADD src/docker-2ndboot.sh  /home/user
+
+ARG NOVNC=0
```

Comment:
This should be runtime environment, not build-time environment variable

Reply:
You're right, this should be 
```Dockerfile
ENV NOVNC=0
```